.github/workflows/lint.yaml

```diff
@@ -0,0 +1,101 @@
+name: Linter
```

Comment:
Can we call the workflow `lint` instead od `linter`?

Reply:
done